airbyte-integrations/connectors/source-amazon-ads/unit_tests/test_source.py

```diff
@@ -64,25 +65,23 @@ def test_check(config_gen):
     assert command_check(source, config_gen(start_date=...)) == AirbyteConnectionStatus(status=Status.SUCCEEDED)
     assert len(responses.calls) == 2
 
-    assert command_check(source, config_gen(start_date="")) == AirbyteConnectionStatus(status=Status.SUCCEEDED)
-    assert len(responses.calls) == 4
+    with pytest.raises(Exception):
+        command_check(source, config_gen(start_date=""))
 
     assert source.check(None, config_gen(start_date="2022-02-20")) == AirbyteConnectionStatus(status=Status.SUCCEEDED)
-    assert len(responses.calls) == 6
+    assert len(responses.calls) == 4
```

Comment:
why did the number of calls change? I couldn't find any related changes in the code

Reply:
we update case in line 67-68 and now we handle wrong pattern, so call qty reduced